src/pdf/ShippingLabels/ShippingLabels.tsx

```diff
@@ -24,8 +27,18 @@ const formatDatetime = (datetimeString: string | null, isDatetime: boolean): str
     return formattedDate;
 };
 
-const getParcelsForDelivery = async (parcelIds: string[]): Promise<Schema["parcels"][]> => {
-    const { data, error } = await supabase.from("parcels").select("*").in("primary_key", parcelIds);
+const getParcelsForDelivery = async (
+    parcelIds: string[]
+): Promise<ParcelWithCollectionCentre[]> => {
+    const { data, error } = await supabase
+        .from("parcels")
+        .select(
+            `*, 
+            collection_centre:collection_centres ( 
+                name
+            )`
+        )
+        .in("primary_key", parcelIds);
```

Comment:
interesting... I didn't know you could match a field to an array of values - I assume this means you're selecting all the rows which has a primary key in parcelIds?

Reply:
Hahaha I think this code was implemented by Paopao but yes basically as long as the key matches one of the many in the list it will be fetched 
[https://supabase.com/docs/reference/javascript/in](https://supabase.com/docs/reference/javascript/in)